src/pages/ptg/index.md

```diff
@@ -18,7 +18,7 @@ header:
   location:
     icon: /img/summit/Vector-world.svg
     text: Virtual
-  buttons:
+  button:
```

Comment:
is this change correct ? from button to buttons ?
check build error https://app.netlify.com/sites/openinfra-dev/deploys/656f2714a855aa04a95fbf5e#L354